tests/integration/views/test_view_submission.py

```diff
@@ -268,6 +268,62 @@ def test_view_submission_does_not_show_parenthesis_trading_as_is_empty_string(se
             self.assertInUrl('view-submission')
             self.assertNotInBody('(Integration Tests)')
 
+    def test_view_submission_default_shows_no_answer_provided(self):
+        no_trading_as_payload = {
+            'user_id': 'integration-test',
+            'period_str': 'April 2016',
+            'period_id': '201604',
+            'collection_exercise_sid': '789',
+            'questionnaire_id': '0123456789000000',
+            'ru_ref': '123456789012A',
+            'response_id': '1234567890123456',
+            'ru_name': 'Integration Testing',
+            'ref_p_start_date': '2016-04-01',
+            'ref_p_end_date': '2016-04-30',
+            'return_by': '2016-05-06',
+            'employment_date': '1983-06-02',
+            'region_code': 'GB-ENG',
+            'language_code': 'en',
+            'roles': [],
+            'trad_as': '',
+            'account_service_url': 'http://upstream.url',
+        }
+        with patch('tests.integration.create_token.PAYLOAD', no_trading_as_payload):
+            self.launchSurvey('test_view_submitted_response')
+
+            # check we're on first page
+            self.assertInBody('What is your favourite breakfast food')
+
+            # We fill in our answer
```

Comment:
Not sure all these comments are necessary

Reply:
Yeah that's copypasta